fahrschein-e2e-test/build.gradle

```diff
@@ -25,5 +25,6 @@ test {
     systemProperty "COMPOSE_PROVIDED", project.hasProperty("e2e.composeProvided") ? "true" : "false"
 }
 
-java.sourceCompatibility = JavaVersion.VERSION_11
-java.targetCompatibility = JavaVersion.VERSION_11
+tasks.withType(JavaCompile).configureEach {
+    options.release = 11
```

Comment:
So setting this option is better in making sure that we go with Java 11 then the options above?

Reply:
The other options are incompatible with gradle toolchains somehow, and this is actually the successor for java9+ and the recommended way going forward.

Reply:
Thanks for the explanation. All clear now. :smile: